.rubocop.yml

```diff
@@ -0,0 +1,21 @@
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  Exclude:
+    - "vendor/**/*"
+    - "tests/integration/**/roles/*"
+    # ignore these until rubocop is incorportaed into qansible
+    - Rakefile
+    - Gemfile
+Style/StringLiterals:
+  # needs agreement
```

Comment:
i do not understand the question. as documented in the issue, it should be either single_quotes or double_quotes.